core/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -346,7 +346,28 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
     OptimizeTableCommand(
       Option(ctx.path).map(string),
       Option(ctx.table).map(visitTableIdentifier),
-      Option(ctx.partitionPredicate).map(extractRawText(_)).toSeq, Map.empty)(interleaveBy)
+      Option(ctx.partitionPredicate).map(extractRawText(_)).toSeq,
+      Map.empty)(interleaveBy)
+  }
+
+  /**
+   * Creates a [[ReorgTable]] logical plan.
+   * Examples:
+   * {{{
+   *   -- Physically delete dropped columns of target table
```

Comment:
dropped columns? -> dropped rows?

Reply:
I changed this to "rows and columns". Columns can be dropped when using column mapping.